skeleton-navigation/src/my-app.css__if_css

```diff
@@ -0,0 +1,48 @@
+body {
+  font-family: 'Source Sans Pro', sans-serif;
+}
+
+nav {
+  background: linear-gradient(135deg, #6d50a2 0%, #ed2b88 100%);
+  display: flex;
+}
+
+a {
+  padding: 10px;
+  text-decoration: none;
+  color: black;
+}
+a:hover {
+  background-color: darkgray;
+}
+a.login, a.user {
+  margin-left: auto;
+}
+
+.footer {
+  background-color: #6d50a2;
+  padding: 10px;
+  color: white;
+  font-size: 10px;
+}
+
+
+.container {
+  display: flex;
+}
+
+
+au-viewport {
+  border: 1px solid lightgray;
+  padding: 10px;
+  flex-grow: 1;
+}
+
+au-viewport:empty {
+  display: none;
+}
+
+
+.goto-active {
+  background-color: lightgray;
+}
```

Comment:
This, however, should remain.

Reply:
> I think what @jwx suggested is that you can remove lots of styles and layout from the skeleton, they were only for the demonstration in his video. We can cut off, and reduce to minium, because end user just want some starting code for router, not layout and styles.

Got it, let me clean up the styling.